tests/solvers/test_cp_local_search_stereo_insertion.py

```
@@ -0,0 +1,2388 @@
+# Multi-run harness tests
+# ---------------------------------------------------------------------------
+
+
+def test_multi_run_different_results_with_rng(solver_imports) -> None:
+    import random
+    from seed import SeedConfig, build_greedy_seed
+    from products import ProductLibrary, ProductSummary
+
+    case = _make_minimal_stereo_case(solver_imports)
+    config = SeedConfig()
+    rng1 = random.Random(1)
```

Ruff (0.15.11)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)